### DISCUSSION TITLE
How to store the results of the calculate_hundegger_machine_data(elements, hundeggertype) method in a variable?

### DISCUSSION THREAD

**Discussion by lucas-ita** [cwapi3d/cwapi3dpython#154] [General]
Hi,

Would it be possible to store the results of the `calculate_hundegger_machine_data(elements, hunderggertype)` method in a variable?
By performing this calculation and accessing the "individual control" option in CadWork, I can obtain information about all elements that come into contact or collide with a specific part. The image below illustrates this process:

![image](https://github.com/cwapi3d/cwapi3dpython/assets/123772761/a0671946-50b5-494a-83ab-ae1d38394d01)

My goal is to use the mentioned calculation method and then obtain the geometries that already contain detailed information about collisions and touches. Later, I plan to group each element with all the pieces that come into contact or collide in a scene.

In summary I would like to take advantage of the existing functionality in Cadwork with the calculate_hundegger_machine_data(elements, hunderggertype) method. If this is not possible, I am aware that it would be necessary to create this logic using the `check_if_elements_are_in_contact(first_element_id, second_element_id)` and `check_if_elements_are_in_collision(first_element_id, second_element_id)` methods.
